pkg/deploy/graph/edges.go

```diff
@@ -73,6 +75,7 @@ func AddDeploymentEdges(g osgraph.MutableUniqueGraph, node *deploygraph.Deployme
 			}
 			if BelongsToDeploymentConfig(node.DeploymentConfig, rcNode.ReplicationController) {
 				g.AddEdge(node, rcNode, DeploymentEdgeKind)
+				g.AddEdge(node, rcNode, ControllerRefEdgeKind)
```

Comment:
The comments I had for you in person remain -- enforcing this weakly by co-locating the two edge creations instead of programmatically by subtyping `ControllerRefEdgeKind` with `DeploymentEdgeKind` is not ideal.